Source/WebCore/platform/gstreamer/GStreamerQuirkWesteros.cpp

```diff
@@ -67,6 +75,13 @@ void GStreamerQuirkWesteros::configureElement(GstElement* element, const OptionS
         g_object_set(element, "caps", defaultCaps.get(), nullptr);
         return;
```

Comment:
I suspect this early return needs to be removed?

Reply:
not really, there are separate invocations of configureElement() for "GstURIDecodeBin3" and for "GstParseBin". For decodebin element there is no need to proceed further as TYPE_NAME won't match other conditions

Reply:
Sorry, I was wrong indeed.